test/test_testing.py

```diff
@@ -5854,3 +5854,34 @@ def test_get_notices(self):
 class TestNotices(unittest.TestCase, _TestingPebbleClientMixin, PebbleNoticesMixin):
     def setUp(self):
         self.client = self.get_testing_client()
+
+
+class TestCloudSpec(unittest.TestCase):
+    def test_get_set_cloud_spec(self):
```

Comment:
Should this be `test_set_cloud_spec`, or similar to above, just `test_set`, given the context is clear from the class name?